Dockerfile

```
@@ -114,7 +123,7 @@ ARG DOCKER_USER
 USER ${DOCKER_USER}
 
 # The default command runs gunicorn WSGI server in the sandbox
-CMD gunicorn -c /usr/local/etc/gunicorn/funmooc.py funmooc.wsgi:application
+CMD gunicorn -c /usr/local/etc/gunicorn/app.py app.wsgi:application
```

```suggestion
ARG SITE
CMD gunicorn -c /usr/local/etc/gunicorn/app.py ${SITE}.wsgi:application
```
Or rename the `sites/${SITE}/src/backend/${SITE}` folder to `app` and `DJANGO_SETTINGS_MODULE` should also follow.

---

yep so we have to enforce the name of this app :man_shrugging: so better force it to something constant like `app`.

---

no.. better call it {SITE} as per Django common practice.